packages/next-auth/src/core/lib/oauth/callback.ts

```diff
@@ -57,7 +54,8 @@ export default async function oAuthCallback(params: {
         profile = JSON.parse(profile)
       }
 
-      return await getProfile({ profile, tokens, provider, logger })
+      const newProfile = await getProfile({ profile, tokens, provider, logger })
+      return { ...newProfile, cookies: [] }
```

Comment:
I believe ignoring the `cookies` here is for security?

Reply:
@lluia This is just for conforming with the check later I think, the logic remains the same:
https://github.com/nextauthjs/next-auth/blob/ffcf2b06d54f096d5d9ad8cd92c63b00210c4b77/packages/next-auth/src/core/routes/callback.ts#L55